server/util/uuid/uuid.go

```diff
@@ -26,6 +27,8 @@ var (
 
 	failsafeID     string
 	failsafeIDOnce sync.Once
+
+	manualHostID = flag.String("executor.host_id", "", "Optional: Allows for manual specification of an executor's host id. If not set, a random UUID will be used.")
```

Comment:
does k8s have an easy way to map these flags to e.g. pod ID? wonder if we should support env vars too like we do for MY_POD_ID etc

Reply:
For env vars, you could do this in your config.yaml
```
executor:
  host_id: ${YOUR_ENVIRONMENT_VARIABLE}
```